assets/js/atomic/blocks/product-elements/button/block.js

```diff
@@ -5,7 +5,10 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { useEffect, useRef } from '@wordpress/element';
-import { useStoreAddToCart, useStoreEvents } from '@woocommerce/base-hooks';
+import {
+	useStoreEvents,
+	useStoreAddToCart,
+} from '@woocommerce/base-context/hooks';
```

Comment:
Could it be that the /hooks index file exports more things that previous, since we moved things from base hooks. Thats the only change I can think of that would impact this.

Reply:
Hmm, perhaps, but then that could mean tree shaking isn't being applied. That would be a good thing to try and determine in a followup (might be even more savings gains there!).